StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Extensions/UIColor+Link.swift

```diff
@@ -17,110 +17,40 @@ extension UIColor {
     /// Brand color for Link.
     ///
     /// Use it as accent color for controls and activity indicators.
-    static let linkBrand: UIColor = UIColor(red: 0.2, green: 0.867, blue: 0.702, alpha: 1.0)
-
-    /// Darker version of the brand color.
-    ///
-    /// Use it as accent color on small UI elements or text links.
-    static let linkBrandDark: UIColor = UIColor(red: 0.020, green: 0.659, blue: 0.498, alpha: 1.0)
+    static var linkBrand: UIColor {
+        LinkUI.useNewBrand ? UIColor(red: 0, green: 0.84, blue: 0.44, alpha: 1.0) : UIColor(red: 0.2, green: 0.867, blue: 0.702, alpha: 1.0)
+    }
 
-    /// Level 600 variant of Link brand color.
+    /// Level 400 variant of Link brand color.
     ///
     /// Use for separator bars over the Link brand color.
-    static let linkBrand600: UIColor = UIColor(red: 0.102, green: 0.773, blue: 0.608, alpha: 1.0)
-
-    /// Main background color.
-    static let linkBackground: UIColor = .dynamic(
-        light: .white,
-        dark: UIColor(red: 0.11, green: 0.11, blue: 0.118, alpha: 1)
-    )
+    static var linkBrand400: UIColor {
+        LinkUI.useNewBrand ? UIColor(red: 0.0, green: 0.64, blue: 0.33, alpha: 1.0) : UIColor(red: 0.102, green: 0.773, blue: 0.608, alpha: 1.0)
+    }
 
     /// Color of the Link logo in the navigation bar.
     static let linkNavLogo: UIColor = .dynamic(
         light: UIColor(red: 0.114, green: 0.224, blue: 0.267, alpha: 1.0),
         dark: .white
     )
 
-    /// Tint color of the nav. Affects the color of nav buttons.
-    static let linkNavTint: UIColor = .dynamic(
-        light: UIColor(red: 0.188, green: 0.192, blue: 0.239, alpha: 1.0),
-        dark: UIColor(red: 0.922, green: 0.922, blue: 0.961, alpha: 0.6)
-    )
-
     /// Color for borders and dividers.
     static let linkSeparator: UIColor = .dynamic(
         light: UIColor(red: 0.878, green: 0.902, blue: 0.922, alpha: 1),
         dark: UIColor(red: 0.471, green: 0.471, blue: 0.502, alpha: 0.36)
     )
```

Comment:
Looks like `linkPrimaryText`, `linkSeparator`, and `linkSecondaryButtonBackground` are only being used by BacsDDMandateview -- maybe we should these over to that view?

Reply:
Good catch!